anti.md

```diff
@@ -1,44 +1,76 @@
 # Anti project
 
 
+
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
 
+
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+The question is, how should users make traditional sketches
+on a computer, as similarly as possible to how they do in real life?
+We could suppose that the result should be in a vector graphic format. 
+For context, I will strongly take the stance that something like a Wacom tablet is not a DSL.
+
+This helps artists in the digital age. It might be that an artist would be somehow able to
+describe their desired artwork, and it would be created.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL is not appropriate for the users in the following sense. Consider a DSL that solves this
+problem. To some extent, it removes the direct interaction that an artist traditionally has
+with their work. There are many styles that benefit from that, but we are specifically considering
+those that don't.
+Any DSL that solves this problem could be made a little better by making it a little less like
+a DSL. The ideal situation would be something where the user drew normally on (perhaps special) paper,
+and the result was recorded. We want to preserve almost all of the interactions the user has
+as they already are.
```

Comment:
What about something similar to a loop pedal but for art instead of music? Some way to draw a shape and then get it to be repeated? There could also be neat animation tools where you draw something and then add more information to get it to do what you want. Even if you don't consider the actual pen-to-surface-drawing a DSL there's lots of other things involved in drawing that we would definitely consider a language. You could even say that what one draws with their pen is a primitive and your language can manipulate these primitives.